yt_dlp/extractor/digiteka.py

```diff
@@ -66,20 +80,28 @@ def _real_extract(self, url):
 
         deliver_info = self._download_json(
             f'http://www.ultimedia.com/deliver/video?video={video_id}&topic={video_type}',
-            video_id)
-
+            video_id,
+        )
```

Comment:
revert the unnecessary formatting change; don't autoformat
```suggestion
            video_id)
```